objects.go

```diff
@@ -16,10 +16,15 @@ type EASearch map[string]interface{}
 
 type EADefListValue string
 
+type MultiObjectResult [][]map[string]interface{}
```

Comment:
Dont use this as a new type . MultiObjectResult can also be a []map[string]interface{}

Reply:
Done